### DISCUSSION TITLE
Welcome to awesome-testflight-link Discussions!

### DISCUSSION THREAD

**Discussion by pluwen** [pluwen/awesome-testflight-link#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by XxNick1995xX** [comment]
Have you added / changed anything to your TestFlight links on your page??

**Reply by pluwen** [reply]
You can view the modification history here:
https://github.com/pluwen/awesome-testflight-link/commits/master

**Reply by eserogta** [reply]
> You can view the modification history here:
> 
> https://github.com/pluwen/awesome-testflight-link/commits/master

link don't work

**Comment by Dandykenn** [comment]
Just purchased  testflight and installed the apps but cant connect to server now i reintstalled apps the. Reinstalled again now need reedem code what ahould i do

**Reply by pluwen** [reply]
Find the testflight url then click it.

**Comment by himia0802** [comment]
Hey everyone!

We would appreciate it if anyone is interested in testing our app called 1001 TVs.  1001 TVs is a user-friendly screen-casting application that offers support for screen-mirroring to a TV, a web browser, and screen-casting to a computer.  

Here is the TestFlight link: https://testflight.apple.com/join/Nf1dKOBG

Feel free to share your feedback at 1001tvs@nero.com

For inquiries, please visit our Instagram page at https://www.instagram.com/tvsnero/


Check out the product webpage at https://www.1001tvs.com/

**Reply by eserogta** [reply]
Im in love, thanks

**Reply by Pictureitgold** [reply]
![image](https://github.com/user-attachments/assets/c74828cb-6065-40c7-9d2b-3b879bc81cb5)

**Comment by Imthebetkale** [comment]
How do I get the invite to TestFlight

**Comment by abbaskabul** [comment]
How to install this app to get updates?

**Reply by himia0802** [reply]
You can get the latest updates on the[ app store](https://apps.apple.com/us/app/screen-mirroring-1001-tvs/id1464125180?pt=1147233&ct=new-1001tvs-web&mt=8).

**Reply by Pictureitgold** [reply]
Yeah but how do you redeem the code it doesn't accept new users from original link?

**Comment by himia0802** [comment]
Hello, and welcome to everyone trying out our exciting new app, Erasee! With Erasee, you can remove backgrounds in just 5 seconds. 

▶Erasee: The AI Background Remover for iOS
You can get a transparent background for any image with just one tap.
#Key Features:
1. Instant Background Removal: With advanced AI, remove backgrounds in 5 seconds.
2. High Precision: Get perfect results, even with tricky details like hair.
3. Multiple Background Options: Save as transparent or add solid colors.
4. Creative Tools: blur backgrounds and add customizable borders. 

Website: https://pcai.nero.com/erasee
Testflight link: https://testflight.apple.com/join/t6xnU9H3
Contact us:  [ai@nero.com](mailto:ai@nero.com)
We greatly appreciate everyone's feedback. Thank you all for your support!

**Reply by CHithubber** [reply]
Elgato is in development for a driver for Stream Deck for iPad: https://github.com/elgatosf/streamdeck-kit-ipad through TestFlight.

Maybe of interest to add this to the list even it is right now in a very early stage and just shows that a Stream Deck unit will be recognized and responding to interaction.

**Comment by CHithubber** [comment]
Hi!This was a beta when I did address it but now after so long time to reply it is now moved over to release.9 nov. 2024 kl. 00:43 skrev Pictureitgold ***@***.***>:﻿
Yeah but how do you redeem the code it doesn't accept new users from original link?

**Comment by Pictureitgold** [comment]
Is okay I’m just confused in the last wallet name their many steps in your
coding that I’m not familiar with.
